src/ParametricFunctionControlPanel.java

```diff
@@ -0,0 +1,215 @@
+import javax.swing.*;
+import javax.swing.text.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class ParametricFunctionControlPanel extends JPanel {
+
+    GridBagLayout ParametricInputPosition = new GridBagLayout();
+    GridBagConstraints gbc = new GridBagConstraints();
+    JTextField functionFieldX = new JTextField();
+    JTextField functionFieldY = new JTextField();
+    JTextField rangeStart = new JTextField();
+    JTextField rangeEnd = new JTextField();
```

Comment:
all of these are missing explicit visibility specifiers

Reply:
Done